ip6.h

```diff
@@ -157,6 +157,7 @@ struct ip6_dest {
 #define IP6OPT_TYPE_ICMP	0xC0
 
 #define IP6OPT_MUTABLE		0x20
+#define IP6OPT_TPF		0x41	/* 01 0 00001 */
```

Comment:
This value has not been assigned by IANA yet (and 0x41 does not follow the registry's guideline for "distinct values in the "rest" subfield", so may not match what is eventually assigned).  I do not think it should be merged with an un-allocated value.